build/postgres/Dockerfile

```diff
@@ -87,6 +88,7 @@ else \
 		tar \
 		bzip2 \
 		lz4 \
+		krb5-workstation \
```

Comment:
🤔 This list doesn't seem to be in any particular order! 🤷🏻

Reply:
Random order has been maintained for now.